.github/scripts/lintrunner.sh

```diff
@@ -31,6 +31,9 @@ python3 -m tools.pyi.gen_pyi \
     --deprecated-functions-path "tools/autograd/deprecated.yaml"
 python3 torch/utils/data/datapipes/gen_pyi.py
 
+# Also check generated pyi files
+find torch -name '*.pyi' -exec git add --force -- "{}" +
```

Comment:
TIL I've learned about `+` variant, don't know when it was the first time I did it, but I would have done it as 
`find torch -name '*.pyi'| xargs git add --force`

Reply:
This is a problem for non-git users (such as myself).